airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -0,0 +1,199 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml
+title: AirbyteProtocol
```

Comment:
this file needed to be renamed and restructured so that java code gen would pick up the new `ConfiguredAirbyteCatalog`. 
* The issue is that the codegen starts from the "main" object in the file and generates all of its dependencies. `ConfiguredAirbyteCatalog` was not a dependency of AirbyteMessage, so it was not being generated. 
* We need `ConfiguredAirbyteCatalog` in this file because it shares an enum with `AirbyteCatalog` and the python code gen can't handle structs with dependencies in different files.